pkg/kubernetes/scripts/projects.js

```diff
@@ -680,7 +819,9 @@
                 kind: "",
                 ocRole: "",
             };
-
+            $scope.itemTracker= function(item) {
+              return item.kind + item.name;
```

Comment:
Doesn't thist need a '/' between the two values? Otherwise there's the distinct risk of the two values flowing into each other and coming up with conflicts.